### PR TITLE
added BasePathLocaleGuesser

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by lunetics** [comment]
Tests are failing in this one

**Comment by lsmith77** [comment]
do you think it makes sense to add this? if yes I can look about fixing the tests

File: Resources/config/guessers.xml [hunk 1]
```diff
@@ -45,10 +45,17 @@
...
         <service id="lunetics_locale.domain_guesser" class="%lunetics_locale.domain_guesser.class%">
             <argument type="service" id="lunetics_locale.validator.meta" />
             <argument type="service" id="lunetics_locale.domain_locale_map" />
             <tag name="lunetics_locale.guesser" alias="domain"/>
+=======
+        <service id="lunetics_locale.base_path_guesser" class="%lunetics_locale.base_path_guesser.class%">
+            <argument type="service" id="lunetics_locale.validator.meta" />
+            <argument>%lunetics_locale.base_path.region_separator%</argument>
+            <tag name="lunetics_locale.guesser" alias="base_path"/>
+>>>>>>> Stashed changes
```
Comment (hacfi) [inline Resources/config/guessers.xml]: Looks like an unresolved merge conflict

**Comment by goetas** [comment]
this is getting old, i'm closing it. if it still relevant, i think that it needs to be re visited

**Pull request closed without merging**